<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagramPrefs</class>
 <widget class="QDialog" name="DiagramPrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diagram Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="fromDateLine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toDateLine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stationTypeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>StationType</string>
   </property>
  </widget>
  <widget class="QLabel" name="stationNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>StationName</string>
   </property>
  </widget>
  <widget class="QLabel" name="viewDataLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>View Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromDateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="toDateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFileButton">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>187</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiagramPrefs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiagramPrefs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
